### ISSUE TITLE
First argument to fetch is optional, but fetch requires at least 1 arg

### ISSUE THREAD

**Issue by JakeTunaley** [Microsoft/TypeScript#25509]
**TypeScript Version:**  3.0.0-dev.20180707

**Search Terms:** fetch lib

**Code**

```ts
fetch();
```

**Expected behavior:**

An error: `Expected 1 arguments, but got 0.`

**Actual behavior:**

No errors are emitted. This is non-compliant with [the `fetch` spec](https://fetch.spec.whatwg.org/#fetch-method), wherein only the second argument is optional.

**Playground Link:** http://www.typescriptlang.org/play/index.html#src=fetch()%3B
